spring-cloud-gcp-autoconfigure/src/test/java/org/springframework/cloud/gcp/autoconfigure/storage/Content.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2017-2018 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Test class for storage.
+ *
+ * @author Raghavan N S
+ * @author Srinivasa Meenavalli
+ */
+
+package org.springframework.cloud.gcp.autoconfigure.storage;
+
+import java.io.Serializable;
+
+public class Content implements Serializable {
```

Comment:
Could you explain the purpose of this new class and why you used it in the storage test?

Reply:
GcpStorageAutoConfigurationTests.java 
Existing code 
		Long actual = testRestTemplate.getForObject("http://localhost:" + this.port + "/resource", Long.class);
		assertThat(actual).isEqualTo(4096);
Error : error: Cannot deserialize instance of `java.lang.Long` out of START_OBJECT token; nested exception is com.fasterxml.jackson.databind.exc.MismatchedInputException: Cannot deserialize instance of `java.lang.Long` out of START_OBJECT toke 

I have created a Pojo( Content) to resolve above issue

Reply:
But that test used to work right? Why did it break in this PR?